src/externs.rs

```diff
@@ -258,3 +258,24 @@ pub unsafe extern "C" fn elog(topic_ptr: *const u8, topic_count: u32, data_ptr:
 		r.borrow_mut().elog(topics, data);
 	});
 }
+
+#[cfg(windows)]
```

Comment:
probably no need [cfg(windows)] here, would be useful for other platforms as well

Reply:
linker on mac/linux probably just silently lets unresolved symbols to make their way to artefacts

Reply:
As you with. I just didn't want to break something on your CI.

Reply:
Thanks! Should be ok